tests/setup/resize_observer.mock.ts

```diff
@@ -6,7 +6,7 @@ class MockResizeObserver {
   observe() {
     //@ts-ignore
     global.resizers.add(this);
-    this.cb();
+    Promise.resolve().then(() => this.cb());
```

Comment:
@LucasLefevre  @dhrp-odoo  the tests seem to pass without it but become problematic in future situations. Do you recall why you had to introducs this?

Reply:
Sorry for the delay; it's been a public holiday week for us.

As specified in the task specs, The issue stemmed from `getFrozenSheetViewRatio` returning NaN and Infinity before mounting, leading to a warning message persisting even after a reload.

To test this behavior, The change was aimed at ensuring a more synchronous behavior in the tests to align with real function behavior.

I got your point; presently, the test passes even without this alteration.
Have there been any changes to related APIs that might have influenced this observed shift in test outcomes?